release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1224,6 +1224,11 @@ This decouples the `intel_pstate` from the `realTime` and `highPowerConsumption`
 
 For pre-IceLake processors, the `intel_pstate` is deactivated by default, whereas for IceLake and later generation processors, the `intel_pstate` is set to `active`.
 
+[id="ocp-4-16-support-for-worker-nodes-on-amd-cpus_{context}"]
+==== Support for Worker Nodes with AMD EPYC Zen 4 CPUs
+
+From release 4.16.30, you can use PerformanceProfiles to configure worker nodes on machines equipped with AMD EPYC Zen 4 CPUs (Genoa and Bergamo). These CPUs are fully supported. However, the power consumption evaluations are ongoing. It is recommended to evaluate features, such as per pod power management, to determine any potential impact.
```

Comment:
```suggestion
From release {product-version}.30, you can use the `PerformanceProfile` parameter to configure compute nodes on machines equipped with AMD EPYC Zen 4 CPUs, such as Genoa and Bergamo. These CPUs are fully supported. However, the power consumption evaluations are ongoing. Consider evaluating features, such as per pod power management, to determine any potential performance impact.
```

Reply:
When I search our docs repo: I see `PerformanceProfile` as a common parameter name for a config file. 

Worker nodes > compute nodes. See https://redhat-documentation.github.io/supplementary-style-guide/#compute-node.

Using the word "recommended" or other variants is not recommended (sorry for the pun) as per the IBM SG.

Reply:
I'm not sure if a link to https://docs.openshift.com/container-platform/4.17/virt/virtual_machines/advanced_vm_management/virt-configuring-default-cpu-model.html would be useful?

Reply:
@dfitzmau thanks Configuring a PerformanceProfile is something you do it is not a parameter, you create a PerformanceProfile and apply it to a cluster

Reply:
Thanks, Kevin. Should it then state "performance profile"?